README.md

```diff
@@ -71,13 +71,20 @@ or by inlining these options in your workflow file.
 
 ### config-file
 
-A string representing the path to an external configuration file. By
-default external configuration files are not used.
+A string representing the path to a configuraton file. It can be a
+local file, or a file located in an external repository. You can use
+this syntax for external repositories: `OWNER/REPOSITORY/FILENAME@BRANCH`.
 
-**Possible values**: A string representing the absolute path to the
-configuration file.
+If the configuration file is located in an external private repository,
+use the [external-repo-token](#external-repo-token) parameter of the
+Action to specify a token that has read access to the repository.
 
-**Example**: `config-file: ./.github/dependency-review-config.yml`.
+**Possible values**: A string representing an absolute path to a file,
+or a file located in another repository:
+
+**Example**: `config-file: ./.github/dependency-review-config.yml # local file`.
```

Comment:
my dumb question, I thought this is a relative path? 🤔

Reply:
Not dumb at all, the example is indeed a relative path! I wrote the docs before we implemented any functionality, will change this!

Reply:
Pushed a fix in 5ea8fbf, don't love it but it's correct. Feel free to change it in the next README PR 😄